### PR TITLE
Added "List" method

### PR DESCRIPTION
**Pull request by Raymond-exe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,4 +5,4 @@
 | Short Desc        | A bot that filters text, images, and video based on user-defined tags.  |
 | Members Committed | Kabir Kwatra, Ivar Rydstrom, Raymond Wong, Aditya Dhir                  |
 | Skills We Got     | TBD                                                                     |
-| Skills Needed     | Also TBD                                                                |
+| Skills Needed     | Also TBD                                                                |
```
Comment (Kab1r) [inline README.md]: remove all changes in this file

File: src/main/java/hackweek/group/filterbot/CommandManager.java [hunk 2]
```diff
@@ -39,10 +42,11 @@ public void handle(MessageReceivedEvent event) {
     private Command getCommand(Message message) {
         String commandPrefix = database.getCommandPrefix(message.getGuild().getId());
         String messageWithoutPrefix = message.getContentStripped().toLowerCase().substring(commandPrefix.length()).trim();
-        if (messageWithoutPrefix.startsWith(Command.HELP.toString().toLowerCase())) return Command.HELP;
-        if (messageWithoutPrefix.startsWith(Command.ADD.toString().toLowerCase())) return Command.ADD;
-        if (messageWithoutPrefix.startsWith(Command.REMOVE.toString().toLowerCase())) return Command.REMOVE;
-        if (messageWithoutPrefix.startsWith(Command.TEST.toString().toLowerCase())) return Command.TEST;
```
Comment (Kab1r) [inline src/main/java/hackweek/group/filterbot/CommandManager.java]: these should not change

File: src/main/java/hackweek/group/filterbot/CommandManager.java [hunk 3]
```diff
@@ -39,10 +42,11 @@ public void handle(MessageReceivedEvent event) {
...
+        if (messageWithoutPrefix.startsWith(Command.HELP.toString())) return Command.HELP;
+        if (messageWithoutPrefix.startsWith(Command.ADD.toString())) return Command.ADD;
+        if (messageWithoutPrefix.startsWith(Command.REMOVE.toString())) return Command.REMOVE;
+        if (messageWithoutPrefix.startsWith(Command.TEST.toString())) return Command.TEST;
+        if (messageWithoutPrefix.startsWith(Command.LIST.toString())) return Command.LIST;
```
Comment (Kab1r) [inline src/main/java/hackweek/group/filterbot/CommandManager.java]: add toLowerCase() to List command toString

File: src/main/java/hackweek/group/filterbot/CommandManager.java [hunk 4]
```diff
@@ -108,10 +113,25 @@ private void test(Message message) {
...
+
+        filters = filters.substring(0, filters.lastIndexOf(","));           //removing the last "," in the "filter" string
+        message.getChannel().sendMessage("Filters in " + message.getGuild().getName() + ": " + filters).queue(); //sends message
+    }
+
     private enum Command {
-        HELP, // Informational Help Command
-        ADD, // Add new Filter
+        HELP,   // Informational Help Command
+        ADD,    // Add new Filter
```
Comment (Kab1r) [inline src/main/java/hackweek/group/filterbot/CommandManager.java]: remove added whitespace

**Review by Kab1r — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Raymond-exe) [reply README.md]: @KabirKwatra how do I do that

Comment (Kab1r) [reply README.md]: I did it for you because it was kinda complicated

File: src/main/java/hackweek/group/filterbot/CommandManager.java [hunk 5]
```diff
@@ -108,10 +113,25 @@ private void test(Message message) {
...
+            filters += tag + ", ";
+
+        filters = filters.substring(0, filters.lastIndexOf(","));           //removing the last "," in the "filter" string
+        message.getChannel().sendMessage("Filters in " + message.getGuild().getName() + ": " + filters).queue(); //sends message
+    }
+
     private enum Command {
         HELP, // Informational Help Command
-        ADD, // Add new Filter
+        ADD,  // Add new Filter
```
Comment (Kab1r) [inline src/main/java/hackweek/group/filterbot/CommandManager.java]: Still an extra whitespace

**Review by Kab1r — APPROVED** [review]

**Pull request merged**